### DISCUSSION TITLE
the iOS app doesn't handle the urls of discussions with specific category

### DISCUSSION THREAD

**Discussion by 3m4r5** [community/community#18130] [Mobile] [answered]
it sends me to safari instead.
example link:
https://github.com/github-community/community/discussions/categories/mobile

**Comment by brianlovin** [comment, marked as answer]
@3m4r5 this might be a regression since the launch of org discussions, I've flagged this for our team to investigate. Thank you!

**Reply by 3m4r5** [reply]
thank you, it also happens when I tap the notification in the app for your answer.
